src/Text/Pandoc/Writers/Docx/OpenXML.hs

```diff
@@ -225,13 +278,23 @@ writeOpenXML opts (Pandoc meta blocks) = do
   toc <- if includeTOC
             then makeTOC opts
             else return []
+  lof <- if includeLOF
+            then makeLOF opts
+            else return []
+  lot <- if includeLOT
+            then makeLOT opts
+            else return []
   metadata <- metaToContext opts
                  (fmap (vcat . map (literal . showContent)) . blocksToOpenXML opts)
                  (fmap (hcat . map (literal . showContent)) . inlinesToOpenXML opts)
                  meta
   let context = resetField "body" body
               . resetField "toc"
                    (vcat (map (literal . showElement) toc))
+              . resetField "lof"
+                   (vcat (map (literal . showElement) lof))
+              . resetField "lot"
+                   (vcat (map (literal . showElement) lot))
```

Comment:
You are setting these variables, but I don't see a change to the default openxml template that adds these variables.  So how do they have any effect?

Reply:
Just found out that the separate `pandoc-templates` repo is a subtree in this repo. Added a commit to address this.